src/SFA.DAS.EpaoRegister/InnerApi/Requests/GetEpaoCoursesRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EpaoRegister.InnerApi.Requests
+{
+    public class GetEpaoCoursesRequest : IGetAllApiRequest
+    {
+        public string EpaoId { get; set; }
+        public string GetAllUrl => $"api/ao/assessment-organisations/{EpaoId}/standards";
```

Comment:
not sure we do everywhere but it would probably be better to have this set in the constructor